README.md

```diff
@@ -1,53 +1,342 @@
+the web fonts implementation that was used in version 4 and older of Font Awesome. You might head over there to learn about how it works.
+
+> https://fontawesome.com/how-to-use/svg-with-js
+
+#### Going from an older pre-release version?
+
+See [UPGRADING.md](./UPGRADING.md).
+
+You might also be interested in the larger umbrella project [UPGRADING.md](https://github.com/FortAwesome/Font-Awesome/blob/master/UPGRADING.md)
+
+## Installation
+
+```
+$ yarn add @fortawesome/fontawesome-svg-core
+$ yarn add @fortawesome/free-solid-svg-icons
+$ yarn add @fortawesome/angular-fontawesome
+```
+
+## Add more styles or Pro icons
+
+Brands are separated into their own style and for customers upgrading from
+version 4 to 5 we have a limited number of Regular icons available.
+
+**Visit [fontawesome.com/icons](https://fontawesome.com/icons) to search for free and Pro icons**
+
+```
+$ yarn add @fortawesome/free-brands-svg-icons
+$ yarn add @fortawesome/free-regular-svg-icons
+```
+
+If you are a [Font Awesome Pro](https://fontawesome.com/pro) subscriber you can install Pro packages.
+
+```
+$ yarn add @fortawesome/pro-solid-svg-icons
+$ yarn add @fortawesome/pro-regular-svg-icons
+$ yarn add @fortawesome/pro-light-svg-icons
+```
+
+Using the Pro packages requires [additional configuration](https://fontawesome.com/how-to-use/js-component-packages).
+
+## Usage
+
+These examples are based on a freshly created project with [Angular CLI].
+
+[Angular CLI]: https://cli.angular.io
+
+### Explicit reference
+
+Not as convenient as using the library but if you believe "explicit is better than
+implicit" then this method if for you.
+
+`src/app/app.component.html`
+
+```html
+<div style="text-align:center">
+  <fa-icon [icon]="faCoffee"></fa-icon>
+</div>
+```
+
+`src/app/app.component.ts`
+
+```typescript
+import { Component } from '@angular/core';
+import { faCoffee } from '@fortawesome/free-solid-svg-icons';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  title = 'app';
+  faCoffee = faCoffee;
+}
+```
+
+`src/app/app.module.ts`
+
+1. Import `{ FontAwesomeModule } from '@fortawesome/angular-fontawesome'`
+1. Add `FontAwesomeModule` to `imports`
+
+```typescript
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+
+import { AppComponent } from './app.component';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  imports: [
+    BrowserModule,
+    FontAwesomeModule
+  ],
+  providers: [],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+
+### Using the Font Awesome Library
+
+The library provides convenient usage in your templates but you have to manage
+the icons separate from your components. This means that if someone
+accidentally removes the icon your component needs your component could break.
```

Comment:
`your component needs your component could break.` -> `your component which uses it could break.` or something